middlewares/validations/validations.js

```diff
@@ -0,0 +1,46 @@
+class Validations {
+  static async validateCreateUser(req, res, next) {
+    const { username, email, password } = req.body;
+    const emailRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
+    const usernameRegex = /^[a-zA-Z0-9]*$/;
+    const pwdRegex = /^(?=.*?[A-Z])(?=(.*[a-z]){1,})(?=(.*[\d]){1,})(?=(.*[\W]){1,})(?!.*\s).{8,}$/;
+
+    switch (true) {
+      case email == null || email === undefined || password == null || password === undefined
```

Comment:
implementation of `==` is wrong because it evaluates the operands values not there types, forexample `77 == '77'` will evaluate to `true`, which should be actually `false` and since `null` is an object in javascript the comparison operation on a `null` will fail. kindly change `==` to triple comparison operator `===`